pandas/core/series.py

```diff
@@ -3702,8 +3703,9 @@ def f(x):
                 mapped = lib.map_infer(values, f, convert=convert_dtype)
 
         if len(mapped) and isinstance(mapped[0], Series):
-            from pandas.core.frame import DataFrame
-            return DataFrame(mapped.tolist(), index=self.index)
+            # GH 25959 use pd.array instead of tolist
```

Comment:
can you remove the pd from above, and instead  just use ``self._constructor_expanddim`` here